jupyter_sphinx/execute.py

```diff
@@ -327,7 +331,8 @@ def apply(self):
         default_kernel = self.config.jupyter_execute_default_kernel
         default_names = default_notebook_names(docname)
         thebe_config = self.config.jupyter_sphinx_thebelab_config
-
+        linenos_config = self.config.jupyter_sphinx_linenos
```

Comment:
As I also wrote in the other place, I think we shouldn't distinguish what was the source of turning line numbers on.

Reply:
This just reads the config option. I could rename it `auto_linenos` if that is clearer. As I note below, I am not necessarily following your point, sorry.

Reply:
I see, looks OK.